charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -38,7 +38,13 @@ defaultRules:
   rules:
     etcd: true
     general: true
-    k8s: true
```

Comment:
Hello @passie ,
Do you have usecase to control those k8s rules separately?
If not, I think having `k8s: true` as the only var to enable&disable all the rules will be more handy to use. 
If yes, we should still reserve `k8s: true` and add sub-rules switch.

Reply:
@Haleygo Yes we do have a use case for it. Currently there is no way to enable or disable the k8s rules if the default rules are enabled. 
`{{- if and .Values.defaultRules.create  }}` 
So the booleans below defaultRules don't do anything if there is no specification regarding the k8s rules. 
I'm happy to change the k8s enabled rules to a group section. Like I had in my first commit. 
So this would something be like
`.Values.k8s.enable` for all the k8s rule files. 
Let me. know your thoughts

Reply:
Set it back to the group rule again.